lib/counter_culture/counter.rb

```diff
@@ -1,6 +1,6 @@
 module CounterCulture
   class Counter
-    CONFIG_OPTIONS = [ :column_names, :counter_cache_name, :delta_column, :foreign_key_values, :touch, :delta_magnitude]
+    CONFIG_OPTIONS = [ :column_names, :counter_cache_name, :delta_column, :foreign_key_values, :touch, :delta_magnitude, :condition ]
```

Comment:
Separate question, but is this change unrelated? I'm a little confused as to the purpose of this.

Reply:
You're right, this looks like another unrelated fix, related with the ability of passing `if:` to `:counter_culture`. @WaKeMaTTa can you add a bit more here, please?

Reply:
in my opinion you can write this as follow: `CONFIG_OPTIONS = %i[ column_names counter_cache_name delta_column foreign_key_values touch delta_magnitude condition ]`